RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -218,7 +223,7 @@ class SiStripClusterizerFromRaw final : public edm::stream::EDProducer<>  {
     std::unique_ptr< edmNew::DetSetVector<SiStripCluster> > 
       output( onDemand ?
 	      new edmNew::DetSetVector<SiStripCluster>(std::shared_ptr<edmNew::DetSetVector<SiStripCluster>::Getter>(std::make_shared<ClusterFiller>(*rawData, *clusterizer_, 
-																	 *rawAlgos_, doAPVEmulatorCheck_)
+																	 *rawAlgos_, doAPVEmulatorCheck_, hybridZeroSuppressed_)
 														       ), 
```

Comment:
some less pedantic indentation less strictly matching each opening brace may make this more readable by fitting the code in the screen width